src/index.js

```diff
@@ -1,2 +1,52 @@
 /* eslint-disable react/react-in-jsx-scope, react/jsx-filename-extension */
 /* @jsx createElement */
+
+function createElement(tagName, props, ...children) {
+  const element = document.createElement(tagName);
+
+  Object.entries(props || {}).forEach(([key, value]) => {
+    element[key.toLowerCase()] = value;
+  });
+
+  children.flat().forEach((child) => {
+    if (child instanceof Node) {
+      element.appendChild(child);
+      return;
+    }
+    element.appendChild(document.createTextNode(child));
+  });
+  return element;
+}
+
+function render(count = 0) {
+  document.getElementById('app').textContent = '';
+  const handleClick = () => {
```

Comment:
핸들러 함수와 구분해주기 위에 빈 줄로 구분해주면 좋을 것 같아요
```suggestion
  document.getElementById('app').textContent = '';
  
  const handleClick = () => {
```